java/ql/lib/ext/java.util.logging.model.yml

```diff
@@ -46,8 +46,15 @@ extensions:
       - ["java.util.logging", "Logger", False, "getLogger", "(String)", "", "Argument[0]", "ReturnValue.SyntheticField[java.util.logging.Logger.name]", "value", "manual"]
       - ["java.util.logging", "Logger", False, "getName", "()", "", "Argument[this].SyntheticField[java.util.logging.Logger.name]", "ReturnValue", "value", "manual"]
       - ["java.util.logging", "LogRecord", False, "LogRecord", "", "", "Argument[1]", "Argument[this]", "taint", "manual"]
+      - ["java.util.logging", "LogRecord", True, "getMessage", "()", "", "Argument[this]", "ReturnValue", "taint", "df-manual"]
   - addsTo:
       pack: codeql/java-all
       extensible: neutralModel
     data:
+      - ["java.util.logging", "Handler", "getEncoding", "()", "summary", "manual"]
       - ["java.util.logging", "Logger", "isLoggable", "(Level)", "summary", "manual"]
+      - ["java.util.logging", "LogRecord", "getResourceBundle", "()", "summary", "df-manual"]
+      # If needed, a pair of manual summary models using synthetics can be made for the two
+      # neutrals below.
+      - ["java.util.logging", "LogRecord", "getParameters", "()", "summary", "manual"]
+      - ["java.util.logging", "LogRecord", "setParameters", "", "summary", "df-manual"]
```

Comment:
If we've by now manually verified that this is just a pojo, then I do think it's relevant to add such manual summary models using synthetic fields. Adding these as manual neutrals just seems plain wrong.

Reply:
There was already a manual neutral for one of the methods - so this was to make this "consistent".
I agree - it is listed in the description as a follow up - I didn't want to also introduce this here as it was complicated enough as it is to investigate the alert discrepancies.

Reply:
Ok.